nameko_sqlalchemy.py

```diff
@@ -22,11 +22,14 @@ def setup(self):
             'service_name': service_name,
             'declarative_base_name': decl_base_name,
         })
+        self.engine = create_engine(self.db_uri)
+
+    def stop(self):
+        self.engine.dispose()
```

Comment:
No. There's no easy way to inspect the engine and verify that it's stopped.

I could write a test which mocks `engine` and verifies that it's called on stop, but it seemed like overkill